### ISSUE TITLE
`IDE0241` is not reported when found attribute

### ISSUE THREAD

**Issue by DoctorKrolic** [dotnet/roslyn#65401]
**Version Used**:
VS 2022 v17.5 Preview 1

**Steps to Reproduce**:
Setup project with nullable enabled in `.csproj`
```cs
#nullable disable

//[Flags]
enum MyEnum
{

}
```
Observer thta `IDE0241` is reported.
Uncomment `//[Flags]`

**Diagnostic Id**:
IDE0241

**Expected Behavior**:
`IDE0241` is still reported because attributes don't create nullability issues

**Actual Behavior**:
`IDE0241` is not reported
